rusqlite_migration/src/asynch.rs

```diff
@@ -80,7 +80,10 @@ impl AsyncMigrations {
     #[allow(clippy::missing_errors_doc)]
     pub async fn current_version(&self, async_conn: &AsyncConnection) -> Result<SchemaVersion> {
         let m = self.migrations.clone();
-        async_conn.call(move |conn| m.current_version(conn)).await
+        async_conn
+            .call(move |conn| Ok(m.current_version(conn)))
+            .await
+            .map_err(Error::from)?
```

Comment:
small improvement I think:
```suggestion
            .await?
```
`?` will take care of the conversion (it calls `into()` behind the scene).